test/CodeGenILTests/IndexingMicroTests.cs

```diff
@@ -186,7 +186,7 @@ public void IndexIntoRange_WithIdent()
 b = 5;
 c=a[b];";
             var ast = ParserUtils.Parse(dscode).Body;
-            var output = codeGen.EmitAndExecute(ast);
+            var output = codeGen.Emit(ast);
```

Comment:
Did you change this to save the assembly to disk? I believe this should continue to be `EmitAndExecute`.

Reply:
yep, just did this while testing, will revert.